ext/hal/Kbuild

```diff
@@ -1,3 +1,7 @@
+# Copyright (c) 2017 Erwin Rol <erwin@erwinrol.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
This header seems inaccurate !

Reply:
Probably just added headers to all open files in my editor that didn't have a header asuming they were new files. 

Remove or change ?

Reply:
Remove please

Reply:
will do.